### PR TITLE
Cucumber admin test

### PR DESCRIPTION
**Pull request by KarenVentura**
Add Integration tests for Admin user

- Sign in
- Sign out
- Sign up
- Admin dashboard

### CODE REVIEW

**Comment by sandrely26** [comment]
Holy @KarenVentura In pipeline your review app is broken. Can you help me doing a pull from master, please?? In master stay the app.json with the postdeploy to run migration. 

I will do it too. I hope everything is fine XD

**Comment by KarenVentura** [comment]
All ready  :+1:

File: app.json [hunk 1]
```diff
@@ -2,7 +2,7 @@
   "name": "ratefast",
   "description": "This app allows you to vote for your favorite meals for the whole week",
   "scripts": {
-    "postdeploy": "bundle exec rake db:setup; db:migrate"
+    "postdeploy": "bundle exec rake db:migrate && bundle exec rake db:setup"
```
Comment (mumoc) [inline app.json]: Why is this change here?

File: features/step_definitions/admin_steps.rb [hunk 2]
```diff
@@ -0,0 +1,310 @@
...
+  create_user
+  sign_in
+  click_link "Admin"
+  click_link "Create new Voting"
+  create_item
+  fill_item_list
+  click_button "Create Voting"
+end
+
+def start_voting
```
Comment (mumoc) [inline features/step_definitions/admin_steps.rb]: like this, use of simple ruby for drying cases ^^

**Comment by mumoc** [comment]
Everything looks good, just the code that doesn't belong here
